go.mod

```diff
@@ -1,12 +1,11 @@
 module github.com/microsoft/moc
 
-go 1.19
```

Comment:
Should we use 1.17 as the version?

Reply:
Just trying to keep it lower, so if other modules are linking they dont fail.
I didnt know moc  operator is also using it